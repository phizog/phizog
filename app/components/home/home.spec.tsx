/* eslint-env jest */
import * as React from 'react'
import { Home } from './index'
import { shallow } from 'enzyme'

describe('<Home />', () => {
  it('should render correctly', () => {
    const component = shallow(<Home />)

    expect(component).toMatchSnapshot()
  })
})
