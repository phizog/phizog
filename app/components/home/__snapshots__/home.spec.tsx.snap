// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Home
    history={[MockFunction]}
    inProgress={true}
    location={[MockFunction]}
    match={[MockFunction]}
    profile={[MockFunction]}
    staticContext={[MockFunction]}
    toggle={[MockFunction]}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Header />,
        <div
          data-tid="container"
        >
          <p>
            This is,
          </p>
          <h2>
            Phizog
          </h2>
          <button
            onClick={[Function]}
          >
            Logout
          </button>
        </div>,
      ],
      "className": "subwindow",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <p>
              This is,
            </p>,
            <h2>
              Phizog
            </h2>,
            <button
              onClick={[Function]}
            >
              Logout
            </button>,
          ],
          "data-tid": "container",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "This is,",
            },
            "ref": null,
            "rendered": "This is,",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Phizog",
            },
            "ref": null,
            "rendered": "Phizog",
            "type": "h2",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Logout",
              "onClick": [Function],
            },
            "ref": null,
            "rendered": "Logout",
            "type": "button",
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Header />,
          <div
            data-tid="container"
          >
            <p>
              This is,
            </p>
            <h2>
              Phizog
            </h2>
            <button
              onClick={[Function]}
            >
              Logout
            </button>
          </div>,
        ],
        "className": "subwindow",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <p>
                This is,
              </p>,
              <h2>
                Phizog
              </h2>,
              <button
                onClick={[Function]}
              >
                Logout
              </button>,
            ],
            "data-tid": "container",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "This is,",
              },
              "ref": null,
              "rendered": "This is,",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Phizog",
              },
              "ref": null,
              "rendered": "Phizog",
              "type": "h2",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Logout",
                "onClick": [Function],
              },
              "ref": null,
              "rendered": "Logout",
              "type": "button",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
